### PR TITLE
Remove GitHub pages and SEEDS- from PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by mchlltt**
## Description:

I think we could do better with this template overall, but I decided to do the minimal, complete no-brainer updates for now: We don't use GitHub pages or the SEEDS project anymore so let's remove them from the template.

## Jira:

-

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -6,15 +6,9 @@
 
 -
 
-## GitHub pages build link:
+## Jira ticket:
```
Comment (iamtheozzy) [inline .github/PULL_REQUEST_TEMPLATE.md:9]:
```suggestion
### Jira ticket:
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -6,15 +6,9 @@
...
-<!--- If you run `yarn docs-publish` locally from the main `seeds` repo, you can publish the github pages docs to your fork to show your changes in a built state. ONLY DO THIS IF YOU HAVE A FORK OF SEEDS. -->
-
-- https://[YOUR GITHUB USER NAME HERE].github.io/seeds/
-
-## Jira:
-
-- [SEEDS-](https://sprout.atlassian.net/browse/SEEDS-)
+- 
 
 ## Screenshots:
```
Comment (iamtheozzy) [inline .github/PULL_REQUEST_TEMPLATE.md:13]:
```suggestion
### Screenshots:
```

**Review by iamtheozzy — COMMENTED** [review]
can we also make these h3's so there's some hierarchy too?

**Comment by mchlltt** [comment]
> can we also make these h3's so there's some hierarchy too?

Done, thanks :)

**Review by iamtheozzy — APPROVED** [review]

**Comment by iamtheozzy** [comment]
Noice! thanks!

**Review by Tathanen — APPROVED** [review]

**Pull request merged**